conftest.py

```diff
@@ -82,3 +90,13 @@ def pytest_configure(config):
             }],
             ALLOWED_HOSTS=['*'],
         )
+
+
+
+for transport in [EventletHTTPTransport,
```

Comment:
OK.  Will probably have to wait a few days before I commit more changes, as I'll be travelling today (to PyCon UK) and I don't have a laptop.